myria/cluster/scripts/cli.py

```diff
@@ -1141,7 +1141,20 @@ def run():
     help="Number of Myria workers per cluster node [default: %d]" % INSTANCE_TYPE_CONFIGS[DEFAULTS['instance_type']].workers_per_node)
 @click.option('--cluster-log-level', show_default=True,
     type=click.Choice(LOG_LEVELS), default=DEFAULTS['cluster_log_level'])
+@click.option('--perfenforce', is_flag=True, help="Enable PerfEnforce (will override default cluster configuration)")
 def create_cluster(cluster_name, **kwargs):
+    # If perfenforce is enabled, override the cluster configuration
+    if kwargs['perfenforce']:
+        click.echo("Adjusting cluster options for PerfEnforce...")
+        kwargs['cluster_size'] = 13
```

Comment:
So with my PerfEnforce implementation, we schedule queries on 4 to 12 workers, hence the number 13. Also, I fixed the instance type because we only have training data for that type of instance. 

This somewhat answers your previous point. We don't want to allow users to override these options if that flag is specified, so I'll add an error for those cases.